kiezcoding.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Kiezcoding @ berlin.js: an all-day co-working event</title>
+    <meta name="description" content="A Meetup for JavaScript developers to socialize and collaborate.">
+    <meta name="keywords" content="Javascript, Usergroup, Berlin, Programming, JS, Client-side, Applications, HTML5, c-working">
+    <meta name="author" content="Berlin.JS">
+    <meta name="viewport" content="width=device-width, target-densitydpi=160dpi, initial-scale=1">
+
+    <link rel="stylesheet" media="screen" href="css/style.css?v=1">
+    <link rel="stylesheet" media="print" href="css/print.css?v=1">
+    <link rel="stylesheet" media="screen" href="css/apps.css?v=1">
+
+    <link rel="stylesheet" media="only screen and (min-width: 480px)" href="css/480.css?v=1">
+    <link rel="stylesheet" media="only screen and (min-width: 768px)" href="css/768.css?v=1">
+    <link rel="stylesheet" media="only screen and (min-width: 992px)" href="css/992.css?v=1">
+    <link rel="stylesheet" media="only screen and (min-width: 1382px)" href="css/1382.css?v=1">
+
+    <link rel="stylesheet" media="only screen and (-webkit-min-device-pixel-ratio: 2), only screen and (min-device-pixel-ratio: 2)" href="css/2x.css?v=1">
+
+    <link href='http://fonts.googleapis.com/css?family=Droid+Sans:400,700' rel='stylesheet' type='text/css'>
+
+    <link rel="apple-touch-icon-precomposed" href="http://berlinjs.org/apple-touch-icon-precomposed.png">
+    <link rel="apple-touch-icon" sizes="72x72" href="http://berlinjs.org/apple-touch-icon-72x72.png">
+    <link rel="apple-touch-icon" sizes="114x114" href="http://berlinjs.org/apple-touch-icon-114x114.png">
+    <link rel="apple-touch-icon" sizes="144x144" href="http://berlinjs.org/apple-touch-icon-144x144.png">
+
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+  </head>
+  <body id="apps">
+    <header role="banner" class="clearfix">
+      <img alt="BERLIN.JS" src="/img/logo.png" class="logo logo-default" />
+      <hgroup>
+        <h4>Berlin.JS Special Event: Kiezcoding</h4>
+        <h1>Daytime co-working pre-Meetup event</h1>
+      </hgroup>
+    </header>
+    <div role="main" class="content clearfix">
+
+      <section class="announcement">
+
+        <p class="intro"><strong>Kiezcoding</strong> is a unique and fun way to change up your regular work routine. You can have the chance to work alongside other friends (and make new ones!) in the Berlin.JS community!</p>
+
+        <p><strong>Join us on Thursday, April 21</strong> at any time, starting from 10:00.</p>
```

Comment:
Maybe here is where we should add the detail "Join us _at co.up_ on Thursday," etc etc?